crates/kitsune_p2p/transport_quic/src/listener.rs

```diff
@@ -51,18 +194,30 @@ impl TransportListenerHandler for TransportListenerQuic {
         Ok(async move { Ok(out) }.boxed().into())
     }
 
-    fn handle_connect(
+    fn handle_create_channel(
```

Comment:
Maybe this one as well. I get the gist that it's taking in some quinn stream and creating the futures channels but I don't really understand the  flow. I think a few little comments would help especially if we are ever chasing bugs through this code in the future.

Reply:
:+1: done